src/chapters/thymeleaf-views/create-template.rst

```diff
@@ -0,0 +1,189 @@
+learn how to implement Thymeleaf templates. To use the library, however, you
+need to provide IntelliJ with some information to make the IDE recognize
+Thymeleaf syntax and commands. You need to include in the proper
+*dependencies*, and there are two common ways to accomplish this.
+
+During Setup for a New Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you create a new Gradle project on the `start.spring.io <https://start.spring.io/>`__
+website, search for and select the *Thymeleaf* dependency. For the practice
+projects you build in this class, be sure to add the *Spring Web* and
+*Spring Boot DevTools* dependencies as well.
+
+.. figure:: ./figures/selectTLdependency.png
+    :alt: Select the Thymeleaf dependency.
+
+In IntelliJ, opening up the ``build.gradle`` file of the new project shows the
+dependencies you selected:
+
+.. sourcecode:: groovy
+   :lineno-start: 22
+
+   dependencies {
+      implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
+      implementation 'org.springframework.boot:spring-boot-starter-web'
+      developmentOnly 'org.springframework.boot:spring-boot-devtools'
+      testImplementation('org.springframework.boot:spring-boot-starter-test') {
+         exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+      }
+   }
+
+Lines 23 - 25 establish links to the thymeleaf, spring-boot-starter-web, and
+spring-boot-devtools libraries, respectively.
+
+Add to an Existing Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you have an existing project that does not currently use Thymeleaf, you
+can add the functionality by updating the ``build.gradle`` file.
+
+.. figure:: ./figures/buildGradleFileTree.png
+   :alt: Open the ``build.gradle`` file.
+
+In the ``dependencies`` block, just paste in the ``implementation`` statement
+seen in line 23 above. Also, be sure to include the Spring Boot libraries if
+the old project is missing those as well.
+
+Start a Practice Project
+-------------------------
+
+Open up your ``hello-spring`` project in IntelliJ and code along with the
+following video.
+
+.. admonition:: Warning
+
+   This video and the others in the chapter walk you through building small
+   web-based projects. Do NOT skip this practice, because the end of chapter
+   exercises pick up where the tutorials end.
+
+.. raw:: html
+
+   <div style="text-align:center;">
+      <iframe width="800" height="450" src="https://www.youtube.com/embed/UIetMLyDVjQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </div>
+
+The sections below outline the main ideas presented in the video. However, the
+text is NOT a substitute for completing the work described in the clip.
+
+Template Location
+^^^^^^^^^^^^^^^^^^
+
+In IntelliJ, store templates in the ``templates`` folder, which you should
+place within the ``resources`` directory. As you will see in the video,
```

Comment:
Confusing phrasing, "In IntelliJ, you should create a ``templates`` folder inside the ``resources`` directory. All of your templates should be put in the ``templates`` folder."

Reply:
Your phrasing is much better.  Edits applied.